src/lib/cli.ts

```diff
@@ -61,6 +69,22 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+type Update = { // eslint-disable-line no-unused-vars
+    latest: string,
+    current: string,
+    type: string,
+    name: string
+};
+
+const notifyNewVersion = (update: Update) => {
+    const changelogUrl: string = `https://sonarwhal.com/about/changelog.html`;
+    // No indentation due to the use of `\` to avoid new line.
+    // https://stackoverflow.com/a/35428171
+    const message: string = `Update available ${chalk.dim(update.current)}${chalk.reset(' → ')}${chalk.green(update.latest)}\nRun ${chalk.cyan(`npm i -g ${update.name}`)} to update\nSee ${chalk.red(changelogUrl)} for update details`;
```

Comment:
> Otherwise I can include both npm i -g and npm i -D o

[`-P |  --save-prod`](https://docs.npmjs.com/cli/install) is also possible. 

> Otherwise I can include both `npm i -g` and `npm i -D` options in the message and ask the user to pick according to their own use.

Not all user will know the differences, so if we do that we will need to also explain the differences between the options, and IMHO that is not the way to go.

My opinion is that the less we asked from the user the better.